docs/source/output.rst

```diff
@@ -66,7 +66,7 @@ tab at the top of the screen. The Overview tab contains links to the:
     * FRASER datasets (fds)
         * Follow the `FRASER vignette <https://www.bioconductor.org/packages/devel/bioc/vignettes/FRASER/inst/doc/FRASER.pdf>`_ for individual FRASER object file (fds) analysis.
     * Results tables
-        * ``results_per_junction.tsv`` this text file contains only significant junctions that meet the cutoffs defined in the config file. 
+        * ``results_per_junction.tsv`` this text file contains only significant junctions that meet the cutoffs defined in the config file. The column ``FDR_set`` specifies on which genes FDR correction was performed (if lists of candidate genes were provided.)
```

Comment:
this is also not correct anymore

Reply:
see above, I removed that sentence again also here